engine/sim/sc_profileset.cpp

```diff
@@ -1115,7 +1123,7 @@ statistical_data_t metric_data( const player_t* player, scale_metric_e metric )
   }
 }
 
-void save_output_data( std::unique_ptr<profile_set_t>& profileset, const player_t* parent_player, const player_t* player, std::string option )
+void save_output_data( profile_set_t* profileset, const player_t* parent_player, const player_t* player, std::string option )
```

Comment:
Would rather make sense to pass it as reference here, and above in the `save_output_call` pass it as a reference too (the `set` parameter).